docs/provider/daemon/getting_started/configure_daemon.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Configure the daemon'
+sidebar_position: 4
+---
+
+The daemon can be configured by exporting environment variables before it is started. It offers many options that let you customize it depending on your needs. 
+
+Here below you can find the list of all environment variables available along with their default values:
+
+* **TDEX_DATADIR** let's you to change the daemon's datadir path, which defaults to `~/.tdex-daemon` for Linux and `~/Library/Application\ Support/Tdex-daemon` for MacOS.
```

Comment:
Can you ash ChatGPT to transform this to a markdown table instead. better visibile